app/controllers/api/v2/aker/jobs_controller.rb

```diff
@@ -9,13 +9,22 @@ class JobsController < ApplicationController
         skip_before_action :verify_authenticity_token
 
         def create
-          @job = ::Aker::Factories::Job.new(params[:job].permit!)
-          if @job.valid?
-
-            @job.create
-            render json: @job, status: :created
-          else
-            render json: @job.errors, status: :unprocessable_entity
+          param_jobs = params[:data]
```

Comment:
I get the fact that you need to create multiple jobs but you are building a list of new jobs, then you are checking whether each of those jobs are valid, then you are creating them and if there is an error then you are looping through them again. How many jobs are you looking to create at a time? Will this create a performance issue. Maybe I am misunderstanding and I don't know what the answer is...